scripts/build-toolchains.sh

```diff
@@ -147,5 +147,5 @@ cd "$RDIR"
 
 # create general env.sh
 echo "# line auto-generated by build-toolchains.sh" >> env.sh
-echo "source \$( realpath \$(dirname "\${BASH_SOURCE[0]:-\${\(%\):-%x}}") )/env-$TOOLCHAIN.sh" >> env.sh
+echo "source $PWD/env-$TOOLCHAIN.sh" >> env.sh
```

Comment:
The printf `%q`  specifier properly quotes all shell metacharacters.

```suggestion
echo "source $(printf '%q' "$PWD")/env-$TOOLCHAIN.sh" >> env.sh
```

Reply:
what does this catch that `echo "source \"$PWD/env-$TOOLCHAIN.sh\""` won't catch?

Reply:
i suppose if $PWD had a `"` in it

Reply:
changed